src/app.js

```diff
@@ -170,6 +171,9 @@ class SidekickLibrary extends LitElement {
     AppModel.appStore.context = this.config;
 
     await loadLibrary();
+
+    // Track library opened
+    sampleRUM('libraryopened');
```

Comment:
```suggestion
    sampleRUM('library:opened');
```

We use colon-separated names for authoring actions and non-separated names for visitor actions.